tasks/install.yml

```diff
@@ -52,8 +58,11 @@
       - craft/storage
       - craft/plugins
       - readme.txt
+      # yamllint disable-line rule:line-length
+  when: craft_info_contents.rc != 0 or craft_version not in (craft_info_contents.stdout_lines[3])
```

Comment:
The `()` around `craft_info_contents.stdout_lines[3]` is not necessary because there are no operators in the expression.